src/views/v2/month/event-multiday.php

```diff
@@ -15,25 +15,55 @@
 
 $event    = $this->get( 'event' );
 $event_id = $event->ID;
+$day_number = $this->get( 'day' );
 
 $classes = [ 'tribe-events-calendar-month__event-multiday' ];
+$classes_inner = [ 'tribe-events-calendar-month__event-multiday-inner' ];
 
-if ( $is_featured = tribe( 'tec.featured_events' )->is_featured( $event_id ) ) {
+// Check if it's featured.
+if ( $is_featured = isset( $event->featured ) && $event->featured ) { // @todo: later use tribe( 'tec.featured_events' )->is_featured( $event_id ) or similar
 	$classes[] = 'tribe-events-calendar-month__event-multiday--featured';
 }
+
+// If it starts today and this week, let's add the left border and set the width
+if ( $starts_today = $event->start_date == $day_number ) { // @todo:later we can check mm/dd or even year
+
+	// @todo: check if it ends this week or not, in order to split the duration
+	$classes[] = 'tribe-events-calendar-month__event-multiday-width-' . $event->duration;
```

Comment:
should be a `--width-2` modifier

Reply:
I'm not sure if I'm following you on this one. We shouldn't be hardcoding here, it comes from the data structure.

Reply:
I understand, I mean the double `--` before the `width` as it is a modifier, I used `--width-2` as an example.

Reply:
oh, I see what you mean now.